example/home/migrations/0002_create_homepage.py

```diff
@@ -53,7 +56,7 @@ class Migration(migrations.Migration):
         [
             ("wagtailcore", "0053_locale_model"),
         ]
-        if wagtail.__version__ >= "2.11"
+        if wagtail_version >= (2, 11)
```

Comment:
you could import https://github.com/wagtail/wagtail/blob/main/wagtail/__init__.py#L10 (`from wagtail import VERSION as WAGTAIL_VERSION`) and then `if WAGTAIL_VERSION >= (2, 11)